index.rst

```diff
@@ -94,12 +94,16 @@ The project has a friendly and growing community. Join our `chat room <https://g
 
 .. toctree::
    :glob:
-   :reversed:
    :caption: Release Notes
    :titlesonly:
    :hidden:
 
-   releases/*
+   v9.0.0 <releases/v9.0.0>
+   releases/v8.0.0
```

Comment:
Why the difference between the first and the following lines here?

Reply:
That's because the file heading (PostgREST version 9.0) and the name that should be shown in the sidebar menu (v9.0.0) are different in this version. For the rest, the names are the same.